model/usersModel.ts

```diff
@@ -0,0 +1,16 @@
+export const joinQuery = ({
+  uid,
+  nickname,
+  myTeam,
+}: {
+  uid: string;
+  nickname: string;
+  myTeam: string;
+}): [string, any[]] => {
+  const sql = `
+        INSERT INTO "BaseballArchive".users (uid, nickname, my_team) 
+        VALUES ($1, $2, $3);
+      `;
+  const values = [uid, nickname, myTeam];
+  return [sql, values];
```

Comment:
return문 전에는 한칸 띄워주시는게 가독성이 좋아서 추천드려요!
```suggestion
  const values = [uid, nickname, myTeam];
  
  return [sql, values];
```